src/OrchardCore.Modules/OrchardCore.Contents/OrchardCore.Contents.csproj

```diff
@@ -22,6 +22,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Feeds.Abstractions\OrchardCore.Feeds.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Indexing.Abstractions\OrchardCore.Indexing.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Liquid.Abstractions\OrchardCore.Liquid.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Localization.Abstractions\OrchardCore.Localization.Abstractions.csproj" />
```

Comment:
Still I dislike this dependency unless we have to. What required you to add it?

Reply:
It is a reference to the Abstractions project, not to the Localization one. 
It is needed to make the T.Plural() work.

Reply:
Ok